pinc/ProofreadingToolbox.inc

```diff
@@ -1,375 +1,88 @@
-        >
-        <input
-            accesskey="]"
-            type="text"
-            value=""
-            name="markBoxEnd"
-            class="dropnormal"
-            size="9"
-            onclick="this.select()"
-        >
-EOS;
-
-        $this->echo_tool_buttons( $round->pi_tools['tool_buttons'] );
-
-        echo '</td>';
-        echo '<td class="top-align right-align">';
-        $help = _("Help");
-        echo "<b><span style='color: red;'>$help---&gt;</span></b>";
-        echo "<a href='../../faq/prooffacehelp.php' accesskey='1' target='helpNewWin'>";
-        echo <<<EOS
-        <img
-            src="gfx/tags/help.png"
-            width="18"
-            height="18"
-            border="0"
-            class="top-align"
-            alt="$help"
-            title="$help"
-        >
-        </a>
-EOS;
-        echo '</td>';
-        echo '</tr>';
-
-        echo '<tr>';
-        echo '<td class="top-align center-align" colspan="3">';
-        echo "<span class='small'>\n";
-
-        echo "<i>"._("Markup shortcuts").":</i>\n";
-        $this->echo_tool_links( $round->pi_tools['tool_links'] );
-        echo "\n<br>\n";
-
-        echo "<i>". _('Pop-up tools'). ":</i>\n";
-        $this->echo_popup_links( $round->pi_tools['popup_links'] );
-        echo "\n<br>\n";
-
-        echo "<i>"._("Reference Information").":</i> ";
-        $url = get_faq_url($round->document);
-        echo "[<a style=\"color:#0000FF; text-decoration: underline;\" href='$url' target='_blank'>";
-        echo _('Guidelines');
-        echo "</a>]\n";
-
-        $url = "$code_url/faq/InterfaceDiagram.png";
-        echo "[<a style=\"color:#0000FF; text-decoration: underline;\" href='$url' target='_blank'>";
-        echo _("Proofreading Diagram");
-        echo "</a>]\n";
-
-        echo "</span>\n";
-        echo '</td>';
-        echo '</tr>';
-        echo '</table>';
-        echo '</form>';
-        echo '</div>';
-    }
-
-
-    // Popup Link
-    // A link that, when clicked, opens a new window containing some tool.
-    private function define_popup_link( $id, $label, $url, $window_name, $width, $height )
-    {
-        $window_features = "width=$width,height=$height,directories=0,location=0,menubar=0,resizable,scrollbars,status=0,toolbar=0";
-        $js = "window.open('$url','$window_name','$window_features'); return false;";
-        $element = "<a href='#' onClick=\"$js\"\n>$label</a>\n";
+// Popup Link
+// A link that, when clicked, opens a new window containing some tool.
+function draw_popup_link($label, $url, $window_name, $width, $height)
+{
+    $window_features = "width=$width,height=$height,directories=0,location=0,menubar=0,resizable,scrollbars,status=0,toolbar=0";
+    $js = "window.open('$url','$window_name','$window_features'); return false;";
+    echo "<a href='#' class='nowrap' onClick=\"$js\"\n>$label</a>\n";
+}
 
-        $this->_popup_links[$id] = $element;
-    }
+function spacer_bar()
+{
+    echo "|\n";
```

Comment:
Make this `echo " | ";`. This ensures that the spacer has a space before the bar, regardless of what's before it, and that it is outputting what is visually seen (a space before and after) rather than a newline (which acts the same in HTML but requires more cognitive load on the developer who looks at this next).